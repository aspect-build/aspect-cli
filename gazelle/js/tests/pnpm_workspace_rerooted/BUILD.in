# gazelle:js_pnpm_lockfile root/pnpm-lock.yaml