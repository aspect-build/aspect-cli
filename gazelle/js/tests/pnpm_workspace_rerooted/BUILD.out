load("@npm//:defs.bzl", "npm_link_all_packages")

# gazelle:js_pnpm_lockfile root/pnpm-lock.yaml

npm_link_all_packages(name = "node_modules")
